config/services.yaml

```
@@ -106,9 +114,9 @@ when@test:
             # See: https://symfony.com/doc/current/service_container/service_decoration.html
             decorates: 'api.adresse.client'
             decoration_inner_name: 'App\Tests\Mock\APIAdresseMockClient::api.adresse.client'
-        App\Tests\Mock\IgnWfsMockClient:
-            decorates: 'ign.wfs.client'
-            decoration_inner_name: 'App\Tests\Mock\IgnWfsMockClient::ign.wfs.client'
+        App\Tests\Mock\BdTopoRoadGeocoderMock:
```

Par rapport à mon commentaire précédent, la subtilité se jouera au niveau de la définition du décorateur : 
```suggestion
        App\Tests\Mock\BdTopoRoadGeocoderMock:
            decorates: 'App\Infrastructure\Adapter\BdTopoRoadGeocoder'
 ```
 Note : la propriété `decoration_inner_name` n'est pas nécessaire